src/components/signup/SignupState.js

```diff
@@ -61,6 +65,7 @@ const Signup = ({ navigation }: { navigation: any, screenProps: any }) => {
     {}
   )
 
+  const [regMethod, setRegMethod] = useState('selfCustody')
```

Comment:
'selfCustody' (and other signup methods) should be defined somewhere as the constants (or enum-like object)

Reply:
done